server/src/main/scala/org/http4s/server/middleware/CORS.scala

```diff
@@ -214,7 +214,7 @@ sealed class CORSPolicy(
 ) {
   import CORSPolicy._
 
-  def apply[F[_]: Functor, G[_]](http: Http[F, G]): Http[F, G] = {
+  def apply[F[_]: Applicative, G[_]](http: Http[F, G]): Http[F, G] = {
```

Comment:
Tests pass, but this breaks binary compatibility.  I don't think we can implement it correctly on Functor.

Reply:
I hacked around this.  Somewhat extreme lengths for a day-old signature, but I like to not make exceptions.